test/e2e/config.go

```diff
@@ -7,4 +7,5 @@ var (
 	infra         = "k8s.gcr.io/pause:3.1"
 	BB            = "docker.io/library/busybox:latest"
 	healthcheck   = "docker.io/libpod/alpine_healthcheck:latest"
+	ImageCacheDir = "/tmp/podman/imagecachedir"
```

Comment:
Consider using `/var/tmp/`and/or checking `$TMPDIR` first to see if there's a preferred location.  I'm assuming the set of cached images is moderately large (hundreds of MB or a GB).   I'm thinking of a case where `/tmp` is a `tmpfs` and therefore competes for memory with actually running tests.  At one point all/most Fedoras had `tmpfs` setup by default, not sure if that's still the case, or similar with other distros.  In any case, it would be nice to have a hood for overriding the location if ever needed.

Reply:
i can perhaps add that as i develop a solution for the remote tests